crates/bevy_render/Cargo.toml

```diff
@@ -9,31 +9,18 @@ license = "MIT OR Apache-2.0"
 keywords = ["bevy"]
 
 [features]
-png = ["image/png", "bevy_image/png"]
-exr = ["image/exr", "bevy_image/exr"]
-hdr = ["image/hdr", "bevy_image/hdr"]
-tga = ["image/tga", "bevy_image/tga"]
-jpeg = ["image/jpeg", "bevy_image/jpeg"]
-bmp = ["image/bmp", "bevy_image/bmp"]
-webp = ["image/webp", "bevy_image/webp"]
-dds = ["ddsfile", "bevy_image/dds"]
-pnm = ["image/pnm", "bevy_image/pnm"]
-
-ddsfile = ["bevy_image/ddsfile"]
-ktx2 = ["dep:ktx2", "bevy_image/ktx2"]
-flate2 = ["bevy_image/flate2"]
-ruzstd = ["bevy_image/ruzstd"]
+# Texture formats (require more than just image support)
 basis-universal = ["dep:basis-universal", "bevy_image/basis-universal"]
+dds = ["bevy_image/dds"]
+exr = ["bevy_image/exr"]
+hdr = ["bevy_image/hdr"]
+ktx2 = ["dep:ktx2", "bevy_image/ktx2"]
```

Comment:
I didn't explicitly check that all of these were actually used by `bevy_render` but I figured that it's good to just keep them all here in case any code gets added that does special-case them. We don't care about generic image storage formats like PNG, but the texture formats might have extra rendering info we need here.